app/utils/api_queries_bag.py

```diff
@@ -19,7 +32,7 @@ def do_bag_search_id(bag_id):
 
 
 @retry(stop=stop_after_attempt(3), after=after_log(logger, logging.ERROR))
-def get_bag_data(uri):
+def get_bag_data_uri(uri):
     """
```

Comment:
Hier klopt de naming ook niet.
Eigenlijk is dit best wel een generieke GET.
Het zou logischer zijn als de BAG url hier in deze functie zou staan, in plaats van die mee te geven.

Reply:
De data wordt opgehaald aan de hand van de `verblijfsobject_url`. Die is dus steeds verschillend. Ik heb de functie hernoemt naar `get_bag_data_by_verblijfsobject_url`

Reply:
Dat is in ieder geval explicieter.